content/blog/2021-04-30-april-21-community-gems.md

```diff
@@ -0,0 +1,183 @@
+
+This likely means that there is an issue with either the credentials or
+permissions set up in WebDav. Make sure that you are using the correct protocol
+and provide a username and/or token or password.
+
+If you use webdavs for your URL and use the `ask_password` parameter for your
+config, this should clear up the issue.
+
+### [What procedures would I use if I needed to remove unwanted or outdated experiments and corresponding data in the cache?](https://discord.com/channels/485586884165107732/485596304961962003/831127462544146482)
+
+You can use `dvc exp gc` with corresponding flags to delete the experiments
+themselves and there are a number of other flags that can be used with
+`dvc exp gc` depending on your needs.
+
+Alternatively, you can remove branches that are no longer required using
+`dvc exp remove`, then run `dvc gc --all-branches --all-tags`. DVC will then go
+through each branch and tag revisions; will keep them, and will purge the rest.
+
+Note that `--all commits` is the safest flag to use, though it will not remove
+much of the cache. Finally, when you delete what's in cache, it does not delete
+the directory completely. It leaves the directory empty in your repo. To find
+info and comment on this known issue, head to
+[Issue #3375 here.](https://github.com/iterative/dvc/issues/3375) Your input is
+ALWAYS valuable to us! 🙏🏼
+
+### [If I have two cloud folder links added to the DVC config, I'm able to push the data to the default one. How could I push the data to the other cloud folder?](https://discord.com/channels/485586884165107732/563406153334128681/833176227762274364)
+
+You're looking for the `-r / --remote` option for `dvc push`. The command looks
+like this:
+
+```
+dvc push --remote name_of_remote_storage
+```
+
+This will upload tracked files or directories to remote storage based on the
+current _dvc.yaml_ and _.dvc_ files.
+
+### [What's the current recommended way to automate hyperparameter search when using DVC pipelines?](https://discord.com/channels/485586884165107732/563406153334128681/829803720190590986)
+
+Take a look at the new
+[experiments feature](https://dvc.org/doc/start/experiments)! It enables you to
+easily experiment with different parameter values.
+
+You could script a grid search pretty easily by queueing an experiment for each
+set of parameter values you want to try. For example,
+`dvc exp run --queue -S alpha={alpha},beta={beta}`, and then compare the results
+with `dvc exp show`.
+
+We are working on developing experiments to have features or documented patterns
+explicitly for grid search support, so definitely share any feedback to help
+drive the future direction of that!
+
+### [How can I pass sensitive strings (connectionString, containerName, etc) to DVC get without storing them to Git?](https://discord.com/channels/485586884165107732/563406153334128681/830021022337073185)
```

Comment:
connectionString -> `connectionString`
containerName -> `containerName`

Reply:
TBH, the title for the question is a bit confusing. It sounds like it's about `--local` config param. In this case it's about a very particular scenario with people importing/getting data. I should be something like:

`When import/get data from a repo how do I provide credentials to the source repo remote storage without saving it into that repo (Git)?`